frontend/workflows/projectCatalog/src/index.tsx

```diff
@@ -55,11 +55,17 @@ const register = (): WorkflowConfiguration => {
         featureFlag: "projectCatalog",
       },
       config: {
```

Comment:
super nit: since we now have to split out a separate entry for the config maybe we should call this something like `configLanding` since it's only used to route into the first config page

Reply:
Sure! I had just thrown in the first thing I thought of to verify it worked, I'll updated it.